konflux-ci/namespace-lister/certificate.yaml

```diff
@@ -0,0 +1,18 @@
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: namespace-lister
+  namespace: namespace-lister
+spec:
+  isCA: true
```

Comment:
Why do we need to be a certificate authority?  We're not going to be issuing certificates of our own.